2-multitenant/modules/env_baseline/main.tf

```diff
@@ -266,6 +266,8 @@ module "gke-standard" {
   enable_private_nodes    = true
   enable_private_endpoint = true
 
+  enable_confidential_nodes = var.enable_confidential_nodes
```

Comment:
https://cloud.google.com/kubernetes-engine/docs/how-to/confidential-gke-nodes#use-in-standard